src/Webfactory/Slimdump/Config/Table.php

```diff
@@ -72,6 +90,14 @@ public function isDataDumpRequired()
     }
 
     /**
+     * @return boolean
+     */
+    public function isAutoIncrement()
```

Comment:
@Dword123 This method name and the name of the setting is not so good IMO. What about something like "keep-auto-increment" or "keep-auto-increment-id" or the like?